src/frontend-pwa/src/App.tsx

```diff
@@ -3,9 +3,9 @@
  * @author Dallas Richmond & Tyler Maloney
  */
 /* eslint-disable no-console */
-import Header from './components/Header/Header';
-import Footer from './components/Footer/Footer';
-import SplashScreen from './components/SplashScreen/SplashScreen';
+import Header from './components/commonComponents/Header/Header';
```

Comment:
I recommend reducing verbosity here. These sub-directories are already within the components directory.

```suggestion
import Header from './components/common/Header/Header';
```

Reply:
Also, if you're creating separate directories on a per component basis, you can simply call the 'main' file within the directory 'index.' When you import you can then just point to the component's directory.

Reply:
@Bottle7 Thanks, should've done that earlier